env/dev/env/util.cljs

```diff
@@ -0,0 +1,16 @@
+(ns env.util
+  (:require [clojure.string :as s]))
+
+(defn get-host [url]
+  (->
```

Comment:
Can you add a comment detailing the expected `url` format? In case the `re-natal` format changes or we want to reuse it. Some tests would be even better !

Reply:
Yup, will do